addons/website_sale/static/src/js/website_sale.js

```diff
@@ -238,7 +238,7 @@ export const WebsiteSale = publicWidget.Widget.extend(VariantMixin, cartHandlerM
         if (!$("#country_id").val()) {
             return;
         }
-        this.rpc("/shop/country_infos/" + $("#country_id").val(), {
+        return this.rpc("/shop/country_infos/" + $("#country_id").val(), {
```

Comment:
it would not hurt if this could be put back in a separate PR as they are in generic.

Reply:
Alright, I have created [this](https://github.com/odoo/odoo/pull/137480) PR. What do you think?